app/controllers/application_controller.rb

```diff
@@ -19,6 +19,16 @@ class ApplicationController < ActionController::API
     Breakers::OutageException
   ].freeze
 
+  VERSION_STATUS = {
```

Comment:
This list feels a bit overly complex to me.. I'd advocate we go with a more simplistic model. Probably more of a conversation than a PR comment though

Reply:
Well this was the PR that I said we'd get to you guys so we could have that conversation, let me know what you'd prefer to see in this case.

Reply:
Here are some thoughts I have, would love other input. I think this could be as simple as

* Draft
* Deprecated

If it's neither of those, it's a normal supported version and the version number tells you current or previous.

Reply:
I reduced it down to 4 options

```
  VERSION_STATUS = {
    dra: 'Draft',
    cur: 'Current',
    pre: 'Previous',
    dep: 'Deprecated'
  }.freeze
```